wordpress-mobile/WordPress-iOS
# Fix username not changing in some situations

Fixes #9179

This fixed the issue of the username not changing on the signup epilogue, in some situations as described in #9179.

To test:
- ensure that username changes happen as they should
- ensure that a username change request doesn't happen when the username is unchanged
